### PR TITLE
Fix/json type

### PR DESCRIPTION
**Pull request by daimeng**
hmm, wasn't able to bootstrap all the examples, only the first 2 are committed.

heres the trace when it crashed in second example:
```
/Users/daimengchen/work/warthog/examples/02-complex-example
$ cd ../.. && yarn && yarn link && cd - && yarn && yarn run clean && yarn db:create && yarn codegen && yarn db:seed:dev
[1/4] 🔍  Resolving packages...
success Already up-to-date.
warning There's already a package called "warthog" registered. This command has had no effect. If this command was run in another folder with the same name, the other folder is still linked. Please run yarn unlink in the other folder if you want to register this folder.
/Users/daimengchen/work/warthog/examples/02-complex-example
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
$ rm -f ./warthog.sqlite.tmp
$ yarn warthog db:create
$ ../../bin/warthog db:create
Database 'warthog-example-2' created!
$ yarn warthog codegen
$ ../../bin/warthog codegen
query: BEGIN TRANSACTION
query: SELECT * FROM "sqlite_master" WHERE "type" = 'table' AND "name" IN ('users')
query: SELECT * FROM "sqlite_master" WHERE "type" = 'index' AND "tbl_name" IN ('users')
query: SELECT * FROM "sqlite_master" WHERE "type" = 'table' AND "name" = 'typeorm_metadata'
creating a new table: users
query: CREATE TABLE "users" ("id" varchar PRIMARY KEY NOT NULL, "created_at" datetime NOT NULL DEFAULT (datetime('now')), "created_by_id" varchar NOT NULL, "updated_at" datetime DEFAULT (datetime('now')), "updated_by_id" varchar, "deleted_at" datetime, "deleted_by_id" varchar, "version" integer NOT NULL, "first_name" varchar(30) NOT NULL, "last_name" varchar(50) NOT NULL, "string_enum_field" varchar NOT NULL, "email" varchar NOT NULL, "registered_at" datetime NOT NULL, "nick_name" varchar(30), "private_field" varchar, "json_field" varying character, CONSTRAINT "UQ_97672ac88f789774dd47f7c8be3" UNIQUE ("email"))
query: COMMIT

/Users/daimengchen/work/warthog/node_modules/ts-node/src/index.ts:261
    return new TSError(diagnosticText, diagnosticCodes)
           ^
TSError: ⨯ Unable to compile TypeScript:
src/modules/user/user.resolver.ts(36,24): error TS2345: Argument of type 'UserCreateInput' is not assignable to parameter of type 'DeepPartial<User>'.
  Types of property 'stringEnumField' are incompatible.
    Type 'string' is not assignable to type 'StringEnum | undefined'.
src/modules/user/user.resolver.ts(45,24): error TS2345: Argument of type 'UserUpdateInput' is not assignable to parameter of type 'DeepPartial<User>'.
  Types of property 'stringEnumField' are incompatible.
    Type 'string | undefined' is not assignable to type 'StringEnum | undefined'.
      Type 'string' is not assignable to type 'StringEnum | undefined'.

    at createTSError (/Users/daimengchen/work/warthog/node_modules/ts-node/src/index.ts:261:12)
    at getOutput (/Users/daimengchen/work/warthog/node_modules/ts-node/src/index.ts:367:40)
    at Object.compile (/Users/daimengchen/work/warthog/node_modules/ts-node/src/index.ts:558:11)
    at Module.m._compile (/Users/daimengchen/work/warthog/node_modules/ts-node/src/index.ts:439:43)
    at Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/Users/daimengchen/work/warthog/node_modules/ts-node/src/index.ts:442:12)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at filePaths.map.fileName (/Users/daimengchen/work/warthog/node_modules/type-graphql/dist/helpers/loadResolversFromGlob.js:10:47)
    at Array.map (<anonymous>)
    at Object.loadResolversFromGlob (/Users/daimengchen/work/warthog/node_modules/type-graphql/dist/helpers/loadResolversFromGlob.js:10:31)
    at options.resolvers.forEach.resolver (/Users/daimengchen/work/warthog/node_modules/type-graphql/dist/utils/buildSchema.js:36:37)
    at Array.forEach (<anonymous>)
```

I'll poke around more in a bit, but posting here just in case you already recognize it.

### CODE REVIEW

**Comment by goldcaddy77** [comment]
@daimeng I actually just fixed all of the examples a few hours ago.  If you rebase from master, you should be able to bootstrap all now.

File: examples/01-simple-model/generated/classes.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { registerEnumType } from "type-graphql";
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const { GraphQLJSONObject } = require("graphql-type-json");
 
-import { BaseWhereInput, PaginationArgs } from "../../../src";
+import { BaseWhereInput, JsonValue, PaginationArgs } from "../../../src";
```
Comment (goldcaddy77) [inline examples/01-simple-model/generated/classes.ts]: Might need to `// @ts-ignore` above this since `JsonValue` is not always used

**Comment by daimeng** [comment]
defaulting to JsonObject (instead of JsonValue) as it's the common case, and what's being generated on the graphql side.

**Pull request merged**